### PR TITLE
fix typo in readme: use span_end not span_pend

### PR DESCRIPTION
**Pull request by tsloughter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ example :
     ...
     Span5 = otter:span_ptag(Span4, "final result", "error"),
     Span6 = otter:span_ptag(Span5, "final result reason", "unknown user"),
-    otter:span_pend(Span6),
+    otter:span_end(Span6),
```
Comment (surik) [inline README.md]: also `otter:span_plog` and `otter:span_ptag` should be changed in the example too

Comment (tsloughter) [reply README.md]: Oops, yup, fixing.

**Comment by tsloughter** [comment]
Note I fixed up the other typos so this should be ready to merge now.

**Comment by cmullaparthi** [comment]
Hi Tristan, really sorry I've just seen this now. Thanks for the pull request but this is now out of date after the 0.2.0 release so I'm going to close it.

**Pull request closed without merging**